Stack/MinStack.swift

```diff
@@ -0,0 +1,35 @@
+class MinStack {
```

Comment:
Could you add the question link and time & space complexity at the top of this file? We usually follow this pattern for every solution.

You can also reference other files for details. Thanks!